arachne-commons/src/main/java/com/odysseusinc/arachne/commons/api/v1/dto/CommonBaseDataSourceDTO.java

```diff
@@ -47,7 +47,7 @@
     private CommonCDMVersionDTO cdmVersion;
 
     private Boolean published;
-
+    @NotNull
```

Comment:
not always is this field notnull. In some cases dto can be used without this field, so should be checked properly

Reply:
it became clear after testing and code research that it doesn't affect field validation in any cases.

Reply:
if it will be ok after testing - maybe add unique constraint in the portal db ?